docs/manual/core-concepts/getters-setters-virtuals.md

```diff
@@ -156,9 +156,9 @@ const user = await User.create({ firstName: 'John', lastName: 'Doe' });
 console.log(user.fullName); // 'John Doe'
 ```
 
-## `getterMethods` and `setterMethods`
+## Deprecated soon: `getterMethods` and `setterMethods`

```

Comment:
Let's just say we deprecate it in v7. Or do we want to keep it for longer?

Reply:
Removal in v7 sounds good. Wanna open a ticket?

Reply:
Woops, I did not read the conversation before mentioning it here. Will make a ticket for it

Reply:
Changed ✔️